src/conq/navigation_lib/waypoint/waypoint_metadata_utils.py

```diff
@@ -0,0 +1,13 @@
+"""This file contains helper functions for encoding and decoding data for waypoints in JSON format.
```

Comment:
Reminder to remove this file if no longer needed

Reply:
Done in [1b889e9](https://github.com/UM-ARM-Lab/conq_python/pull/52/commits/1b889e97cfdd1818c53ea752909ff275f476d662)